my-app/src/app/courses-page/course-item/course-item.component.css

```diff
@@ -0,0 +1,177 @@
+:host {
+    display: flex;
+    justify-content: space-between;
+    height: auto;
+    border-radius: 6px;
+    background: var(--white);
+    box-shadow: 0 1px 2px 0 rgba(29, 30, 38, 0.1), 0 2px 10px 0 rgba(29, 30, 38, 0.15);
+    padding: 24px;
+    margin-bottom: 24px;
```

Comment:
since margin affects other components it's better to define such thing as a part of the layout of the parent component, where this component is used